### PR TITLE
Add the `xz_utils` build requirement

### PR DESCRIPTION
**Pull request by bysnack**
To be able to use the `lzma` python package

### CODE REVIEW

**Comment by dean0x7d** [comment]
Can you bump the version to 1.3.3 and add an entry to the changelog? You can put in today's date and we can release it.

**Comment by bysnack** [comment]
> Can you bump the version to 1.3.3 and add an entry to the changelog? You can put in today's date and we can release it.

Done!

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v1.3.3 | 2021-09-30
+
+- Added `xz_utils` to the list of build requirements
```
Comment (dean0x7d) [inline changelog.md:5]:
```suggestion
- Fixed missing `lzma` module on some Linux systems (added `xz_utils` to the list of build requirements)
```

Comment (bysnack) [reply changelog.md]: Way better, changed.

**Review by dean0x7d — APPROVED** [review]
Great, thanks!

**Pull request merged**